mods/ra/maps/monster-tank-madness/map.yaml

```diff
@@ -2101,6 +2101,7 @@ Rules:
 		-ConquestVictoryConditions:
 		MissionObjectives:
 			EarlyGameOver: true
+		EnemyWatcher:
 	World:
 		-CrateDrop:
```

Comment:
Seems like you're already doing so: https://github.com/OpenRA/OpenRA/pull/8000/files#diff-b3c75601f9a3ffd7cb39ffa245591e5cR2105